src/jit/bitsetasshortlong.h

```diff
@@ -513,11 +513,10 @@ class BitSetOps</*BitSetType*/ BitSetShortLongRep,
 #endif
 
                 // If there's a bit, doesn't matter if we're short or long.
-                if (hasBit)
+                if (m_bits != 0)
                 {
```

Comment:
maybe you could also `assert(hasBit)` 

---
In reply to: [122267573](https://github.com/dotnet/coreclr/pull/11945#discussion_r122267573) [](ancestors = 122267573)